tests/inspectdb/tests.py

```diff
@@ -101,10 +101,8 @@ def test_number_field_types(self):
         assertFieldType('bool_field', "models.{}()".format(bool_field_type))
         assertFieldType('null_bool_field', 'models.{}(blank=True, null=True)'.format(bool_field_type))
 
-        if connection.features.can_introspect_decimal_field:
-            # django-spanner creates DecimalField as FLOAT64 until Spanner
-            # supports numeric.
-            assertFieldType('decimal_field', "models.FloatField()")
+        if connection.vendor != 'sqlite':
+            assertFieldType('decimal_field', "models.DecimalField(max_digits=6, decimal_places=1)")
```

Comment:
Looks like I'm still seeing an error in `inspectdb.tests.InspectDBTestCase.test_number_field_types`:

```
======================================================================
FAIL: test_number_field_types (inspectdb.tests.InspectDBTestCase)
Test introspection of various Django field types
----------------------------------------------------------------------
Traceback (most recent call last):
  File ".../django/tests/inspectdb/tests.py", line 108, in test_number_field_types
    assertFieldType('decimal_field', "models.DecimalField(max_digits=6, decimal_places=1)")
  File ".../django/tests/inspectdb/tests.py", line 56, in assertFieldType
    self.assertEqual(definition, out_def)
AssertionError: 'mode[22 chars]gits=6, decimal_places=1)' != 'mode[22 chars]gits=10, decimal_places=5)  # max_digits and d[77 chars]loat'
- models.DecimalField(max_digits=6, decimal_places=1)
+ models.DecimalField(max_digits=10, decimal_places=5)  # max_digits and decimal_places have been guessed, as this database handles decimal fields as float
```

Reply:
I am seeing the same error, wanted to check with you, what should we do if someone tries to create a decimal field with custom precision and decimal places ? should we throw an error or overwrite to spanner precission and scale.

Reply:
for now fixed it by changing the verification logic to use the same as sqllite.